ViewModel/Saving/PersistenceFactory.cs

```diff
@@ -83,6 +84,14 @@ public static IElementManager GetElementManager(Utilities.ChildElement element)
             {
                 return new IASPersistenceManager(StudyCacheForSaving);
             }
+            else if(element.GetType() == typeof(Alternatives.AlternativeElement))
```

Comment:
This should be switched over to return one output variable that's defaulted to null, instead of each case returning something.

Reply:
Also unnecessary namespacing I think.  unless there's another AlternativeElement class, then this is ok